api/mesh/v1alpha1/mesh.proto

```diff
@@ -131,6 +135,14 @@ message CertificateAuthorityBackend {
   // Mode defines the behaviour of inbound listeners with regard to traffic
   // encryption
   Mode mode = 5;
+
+  // RootCert defines settings related to CA root certificate chain.
+  message RootCert {
```

Comment:
I don't think `RootCert` is correct. This can be intermediate + root cert as stated in the comment. Maybe `RootChain` ? `RootCertsChain` ? `CABundle`?

Reply:
sounds good to me